2019-07-23-intro/animal/Bird/Eagle/index.js

```diff
@@ -0,0 +1,49 @@
+import {Bird} from "../";
+
+/**
+ * Describes an animal belonging to the species Eagle.
+ */
+export class Eagle extends Bird {
+    /**
+     * Gets a description of the animal as parameters.
+     * @param {string} species - species of animal.
+     * @param {Number} numberOfLegs - the number of paws of the animal.
+     * @param {boolean} vegetarian - animal s vegetarian or not.
+     */
+    constructor(species, numberOfLegs, vegetarian) {
+        super(species, numberOfLegs, vegetarian);
```

Comment:
We shouldn't receive species and numberOfLegs in Eagle constructor. It is already known.